### PR TITLE
Pinned posts

### PR DESCRIPTION
**Pull request by JustRoxy**

### CODE REVIEW

**Comment by JustRoxy** [comment]
oh yes, how tf do I need to make this thing to not include previous commits on previous branch 🤕

**Comment by stanriders** [comment]
> oh yes, what tf do I need to do so the thing do not include previous commits on previous branch 🤕

Force push / merge master

File: backend/CircleForms/Services/Database/PostRepository.cs [hunk 1]
```diff
@@ -181,6 +181,28 @@ public async Task AddAnswer(string postId, Answer entry)
...
+    {
+        var redisDb = _redis.GetDatabase();
+        var key = $"post:{postId}";
+
+        if (!redisDb.KeyExists(key))
+        {
+            return false;
+        }
+
+        await redisDb.SetAddAsync("posts:pinned", key);
```
Comment (stanriders) [inline backend/CircleForms/Services/Database/PostRepository.cs:194]: Do we not want to store pinned-ness in the mongo as well?

**Review by stanriders — CHANGES_REQUESTED** [review]

File: backend/CircleForms/Services/Database/PostRepository.cs (hunk 1 above)
Comment (JustRoxy) [reply backend/CircleForms/Services/Database/PostRepository.cs:194]: i dont think so, probably we can add 
`public bool IsPinned { get; set; }` 
property, but it seems quite redundant atm

Comment (JustRoxy) [reply backend/CircleForms/Services/Database/PostRepository.cs:194]: later on it should be done, but not right now

Comment (stanriders) [reply backend/CircleForms/Services/Database/PostRepository.cs:194]: sure

**Pull request merged**